services/user_service.py

```
@@ -0,0 +1,292 @@
+    user.refresh_token_expires_at = now_kst + datetime.timedelta(days=REFRESH_TOKEN_EXPIRE_DAYS)
+    await db.commit()
+    
+    return Token(access_token=new_at, token_type="bearer"), new_rt
+
+async def logout(db: AsyncSession, user: User):
+    """DB에서 RT 정보를 삭제하여 무효화합니다."""
+    user.refresh_token_hash = None
+    user.refresh_token_expires_at = None
+    await db.commit()
+
+async def withdraw_user(db: AsyncSession, user: User, kakao_access_token: str | None = None):
```

Ruff (0.15.0)

[warning] Unused function argument: `kakao_access_token`

(ARG001)